contrib/opentimelineio_contrib/adapters/tests/test_srt_adapter.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright Contributors to the OpenTimelineIO project
+#
+# Licensed under the Apache License, Version 2.0 (the "Apache License")
+# with the following modification; you may not use this file except in
+# compliance with the Apache License and the following modification to it:
+# Section 6. Trademarks. is deleted and replaced with:
+#
+# 6. Trademarks. This License does not grant permission to use the trade
+#    names, trademarks, service marks, or product names of the Licensor
+#    and its affiliates, except as required to comply with Section 4(c) of
+#    the License and to reproduce the content of the NOTICE file.
+#
+# You may obtain a copy of the Apache License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the Apache License with the above modification is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the Apache License for the specific
+# language governing permissions and limitations under the Apache License.
+#
+
+"""Unit tests for the srt file adapter"""
+
+import unittest
+import os
+import tempfile
+
+import opentimelineio as otio
+
+OTIO_SAMPLE_DATA_DIR = os.path.join(
+    os.path.dirname(__file__),
+    "sample_data"
+)
+SRT_EXAMPLE_PATH = os.path.join(OTIO_SAMPLE_DATA_DIR, "srt_example.srt")
+SRT_OTIO_EXAMPLE_PATH = os.path.join(OTIO_SAMPLE_DATA_DIR,
+                                     "srt_example.otio")
+
+
+class SRTTest(unittest.TestCase):
+    def setUp(self):
+        fd, self.tmp_path = tempfile.mkstemp(suffix=".srt", text=True)
```

Comment:
Hi @KarthikRIyer!
It might be a long shot, but I recognized an error when comparing multi-lined strings. Try adding `maxDiff = None` as a class variablle in `SRTTest`
```suggestion
class SRTTest(unittest.TestCase):
    maxDiff = None
    
    def setUp(self):
        fd, self.tmp_path = tempfile.mkstemp(suffix=".srt", text=True)
```

Reply:
This did't work.

maxDiff is just the max length of Diff for the compare function right?

Reply:
Yes. I've had to use it in other tests to make them pass. The error looked similar.